dev-packages/application-manager/src/generator/webpack-generator.ts

```diff
@@ -54,7 +55,7 @@ const monacoHtmlLanguagePath = '${this.resolve('monaco-html', 'release/min')}';`
 module.exports = {
     entry: path.resolve(__dirname, 'src-gen/frontend/index.js'),
     output: {
-        filename: 'bundle.js',
+        filename: '[chunkhash].js',
```

Comment:
Wouldn't it be better and a bit more clear to keep the `bundle` name before the hash suffix, like this:
```
bundle-[chunkhash].js
```
?

Reply:
You are right, I just thought that its too long. However, i think its preferrable to slice the `chunkhash` length to like 8. So it becomes 
```
bundle-[chunkhash:8].js
```
This way it is much clearer specially when debugging.